.github/workflows/release.yaml

```
@@ -20,7 +23,10 @@ jobs:
       - name: Release
         uses: goreleaser/goreleaser-action@90a3faa9d0182683851fbfa97ca1a2cb983bfca3 # v6
         with:
-          version: v1.26.2
+          # You can pass flags to goreleaser via GORELEASER_ARGS
+          # --clean will save you deleting the dist dir
           args: release --clean
+          distribution: goreleaser # or 'goreleaser-pro'
+          version: "~> v2" # or 'latest', 'nightly', semver
         env:
-          GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

AFAIK, GitHub Action secrets are not allowed to be prefixed with `GITHUB_`
```suggestion
          GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```